src/main/scala/view/graphicUI/GraphicAddQuizMenu.scala

```diff
@@ -38,7 +36,7 @@ class GraphicAddQuizMenu(stage: Stage) extends GraphicView:
   var questionTextField: TextField = _
 
   @FXML
-  var scoreIntegerField: IntegerField = _
+  var scoreIntegerField: TextField = _
```

Comment:
perchè ampliarlo a TextField?

Reply:
IntegerField non esite in scene builder, causando errori ed impedendo di aprire l'FXML

Reply:
non esiste un equivalente? 
altimenti se uno inserisce una stringa cause errori nel programma vero e proprio